xen/arch/arm/domain.c

```diff
@@ -561,6 +561,12 @@ void free_vcpu_struct(struct vcpu *v)
     free_xenheap_pages(v, get_order_from_bytes(sizeof(*v)));
 }
 
+uint32_t get_pmu_counters(void)
```

Comment:
I am not sure if this function should be defined in `domain.c`. Moreover - without `static` attribute. On other hand I can't find suitable place for it.

Maybe it is worth to drop it altogether? Just  use

```
v->arch.mdcr_el2 |= READ_SYSREG(PMCR_EL0) >> ARMV8_PMU_PMCR_N_SHIFT) &
           ARMV8_PMU_PMCR_N_MASK;
```

Reply:
Oh, yes about the static - it's my fault. I agree that it's better to change the code as you proposed.